hack/.golint_failures

```diff
@@ -360,6 +360,7 @@ pkg/volume/scaleio
 pkg/volume/storageos
 pkg/volume/testing
 pkg/volume/util/fs
+pkg/volume/util/nsenter
```

Comment:
How hard would be to make lint happy here?

Reply:
I'm sure it wouldn't be hard. Mostly adding properly added comments to exported items.

I was trying to minimize changes with this PR, and mostly just move things as-is. Okay if I take that task on as a separate PR?

Reply:
though I could quickly add it as a separate commit on this PR, and that would remove the need for an approval from `hack/OWNERS`...